src/tokenizer.ts

```diff
@@ -128,6 +128,15 @@ export class Tokenizer implements Tokens.Source {
     return {token, str: this.buffer.group()!};
   }
 
+  private csStrTok(): Token {
```

Comment:
(nit) Do you mind calling this `csTok`?  It feels a little more consistent to me, but I don't feel strongly.